src/navigation/SwipeNavigator.js

```diff
@@ -34,20 +34,22 @@ import { Box, Columns, Stack } from '@/design-system';
 import { ButtonPressAnimation } from '@/components/animations';
 
 import logger from 'logger';
+import PointsScreen from '@/screens/PointsScreen';
 import WalletScreen from '@/screens/WalletScreen';
 import RecyclerListViewScrollToTopProvider, {
   useRecyclerListViewScrollToTopContext,
 } from '@/navigation/RecyclerListViewScrollToTopContext';
 import { discoverOpenSearchFnRef } from '@/screens/discover/components/DiscoverSearchContainer';
 import { InteractionManager, View } from 'react-native';
 import { IS_DEV, IS_IOS, IS_TEST } from '@/env';
+import config from '@/model/config';
 import SectionListScrollToTopProvider, {
   useSectionListScrollToTopContext,
 } from './SectionListScrollToTopContext';
 
-const NUMBER_OF_TABS = 3;
+const HORIZONTAL_TAB_BAR_INSET = 6;
```

Comment:
will need to get a look at this on android @ibrahimtaveras00 can be in release regression

Reply:
Looks pretty solid on my end. Tested both on a Pixel and my Nothing phone 2
<img width="487" alt="Screenshot 2023-10-20 at 10 52 14 AM" src="https://github.com/rainbow-me/rainbow/assets/6284525/19564716-dbb9-48ce-a5e5-68299df41c7e">